src/wp-admin/includes/class-wp-media-list-table.php

```diff
@@ -77,13 +77,15 @@ public function prepare_items() {
 		 * See File_Upload_Upgrader class.
 		 */
 		$not_in = array();
-
-		foreach ( _get_cron_array() as $cron ) {
-			if ( isset( $cron['upgrader_scheduled_cleanup'] ) ) {
-				$details = reset( $cron['upgrader_scheduled_cleanup'] );
-
-				if ( ! empty( $details['args'][0] ) ) {
-					$not_in[] = (int) $details['args'][0];
+		$crons  = _get_cron_array();
+		if ( is_array( $crons ) ) {
+			foreach ( $crons as $cron ) {
```

Comment:
Yes, I agree with this implementation over type casting to `(array)` within the `foreach`. I also agree with checking for presence of type `array` rather than the failed state of `false`.